<launch>
  <node name="lidar_decoder" pkg="slam" type="lidar_decoder.py" />
  <node name="object_detector" pkg="slam" type="clustering.py" />
</launch>
